ci-operator/step-registry/hypershift/mce/agent/metal3/conformance/cilium/hypershift-mce-agent-metal3-conformance-cilium-workflow.yaml

```diff
@@ -0,0 +1,60 @@
+workflow:
+  as: hypershift-mce-agent-metal3-conformance-cilium
+  documentation: |-
+    The HyperShift mce agent conformance (connected) workflow executes
+    tests against a new ephemeral cluster with Cilium CNI installed.
+    The installation method for nodepool is metal3.
+    The current workflow supports both IPv4 and dual-stack
+  steps:
+    allow_best_effort_post_steps: true
+    allow_skip_on_success: true
+    post:
+    - chain: hypershift-dump
+    - chain: hypershift-mce-agent-destroy
+    - chain: baremetalds-ipi-post
+    test:
+    - ref: hypershift-mce-agent-info
+    - chain: hypershift-conformance
+    pre:
+    - ref: baremetalds-packet-setup
+    - ref: cilium-conf
+    - ref: ipi-install-rbac
+    - ref: baremetalds-devscripts-proxy
+    - ref: ipi-install-hosted-loki
+    - ref: baremetalds-devscripts-setup
```

Comment:
If change to
```
- chain: baremetalds-ipi-pre
- ref: cilium-conf
```
can be work? im not sure

Reply:
No. cilium-conf must be called before baremetalds-devscripts-setup so that it properly configures the installation.